python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -603,16 +603,16 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.address_family_evpn.bgp_additional_paths.receive is arista.avd.defined(true) %}
       bgp additional-paths receive
 {%         endif %}
-{%         if router_bgp.address_family_evpn.bgp_additional_paths.send.any is arista.avd.defined(true) %}
-      bgp additional-paths send any
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.backup is arista.avd.defined(true) %}
-      bgp additional-paths send backup
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp is arista.avd.defined(true) %}
-      bgp additional-paths send ecmp
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit is arista.avd.defined %}
-      bgp additional-paths send ecmp limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit }}
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.limit is arista.avd.defined %}
-      bgp additional-paths send limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.limit }}
+{%         if router_bgp.address_family_evpn.bgp_additional_paths.send is arista.avd.defined %}
+{%             set evpn_add_path = 'bgp additional-paths send' %}
+{%             if router_bgp.address_family_evpn.bgp_additional_paths.send_ecmp_limit is arista.avd.defined and router_bgp.address_family_evpn.bgp_additional_paths.send == 'ecmp' %}
+{%                 set evpn_add_path =  evpn_add_path ~ ' ecmp limit ' ~ router_bgp.address_family_evpn.bgp_additional_paths.send_ecmp_limit %}
+{%             elif router_bgp.address_family_evpn.bgp_additional_paths.send_limit is arista.avd.defined and router_bgp.address_family_evpn.bgp_additional_paths.send == 'limit' %}
+{%                 set evpn_add_path = evpn_add_path ~ ' limit ' ~ router_bgp.address_family_evpn.bgp_additional_paths.send_limit %}
+{%             else %}
```

Comment:
instead of adding this else you can give it in line 607
same as suggested here https://github.com/aristanetworks/avd/pull/3730#discussion_r1741481780

Reply:
also, need to add condition for `disabled` here as well

Reply:
Not sure if we need disabled everewhere. Previously it is only added in address_family_evpn

Reply:
Sorry, I meant inside `neighbors` at other places like address_family_path_selection etc
```
router bgp 6500
   address-family path-selection
      no neighbor 1.1.1.1 additional-paths send any
```

Reply:
May be we should not add negation commands. It was added only for evpn. May be it was for any purpose or can we remove it?